### DISCUSSION TITLE
Second data connection losing track of exported file upon changing the model

### DISCUSSION THREAD

**Discussion by AndreasLiiv** [Spine-project/Spine-Toolbox#2664] [Support]
We are rather new to spine and are trying to implement exported data from one data store in to another. But whenever any changes are made to the model the second data connection gives us the error message "Cannot find some file references. Please, check that the files exist" for the excel file generated by the exporter. This can easily be solved by refreshing the references in the data connection and running the program again. This becomes a problem once you want to implement this sequence of components with several exports and data connections in one model.

We are currently trying to model a hydro power system where each of the power plants optimize their electricity production without regards to any of the other hydro plants. Is there another way of doing this in which this error can be avoided? 

![image](https://github.com/spine-tools/Spine-Toolbox/assets/159267461/28053737-074b-4d09-926d-22a18d691a41)

**Comment by ptsavol** [comment]
Delete `Data Connection 2` and connect the `Exporter` directly to `Importer 22` and `Importer 21`. Exporter -> Data Connection links do not do anything.

**Reply by AndreasLiiv** [reply]
Thank you for you quick response! That solved the issue
